apiserver/cmd/main.go

```diff
@@ -144,7 +144,17 @@ func startHttpProxy() {
 	topMux.HandleFunc("/healthz", serveHealth)
 	serveSwaggerUI(topMux)
 
-	if err := http.ListenAndServe(*httpPortFlag, topMux); err != nil {
+	// Create a custom HTTP server with timeouts.
```

Comment:
Is this a linter change? If not, would you mind explaining why this change is needed?

Reply:
Yes, here's linter complaint
```sh
cmd/main.go:147:12: G114: Use of net/http serve function that has no support for setting timeouts (gosec)
        if err := http.ListenAndServe(*httpPortFlag, topMux); err != nil {
                  ^
```

Reply:
The complain is asking for timeout, so after my change in the PR, it no longer complains any more.
I also start an issue to configure timeout properly: https://github.com/ray-project/kuberay/issues/3306